src/config.py

```diff
@@ -155,6 +157,18 @@ def valid_secret(cls, value):
             raise ValueError("Invalid secret, must be a valid base64 encoded json") from e
         return value
 
+    @validator("cpu")
+    def valid_cpu_input(cls, value):
+        if not isinstance(value, float):
+            raise ValueError(f"Invalid CPU input: '{value}', should be of type 'float'")
+        return value
+
+    @validator("memory")
+    def valid_memory_input(cls, value):
+        if not isinstance(value, float):
+            raise ValueError(f"Invalid Memory input: '{value}', should be of type 'float'")
+        return value
+
```

Comment:
Sorry to mislead you in a previous comment, but after testing `pydantic` a bit, it does this validation for you (and even prints a nice error message 🤯 ), so we can remove these validators, (and possibly some of the other ones as well - but that is out of the scope for this PR!)
```suggestion
```

Reply:
<img width="506" alt="Screenshot 2021-02-15 at 12 31 54" src="https://user-images.githubusercontent.com/8521241/107941305-d0d8bd80-6f89-11eb-8189-f3677ce68ee2.png">

Reply:
Ah this is really nice!